src/Pages/ProductList/ProductList.js

```diff
@@ -1 +1,32 @@
 import React, { Component } from "react";
+import Header from "./Components/Header/Header";
+import Category from "./Components/Category/Category";
+import CardList from "./Components/CardList/CardList";
+import "./ProductList.scss";
+
+class ProductList extends Component {
+  state = {
+    products: [],
+  };
+
+  //데이터로딩
+  componentDidMount() {
+    fetch("http://localhost:3000/Data/ProductList.json")
```

Comment:
포트번호는 생략가능합니다! 로컬호스트 포트 번호가 바뀔경우 모든 fetch 주소를 수정해주어야하기때문에 local host는 아래와 같이 생략하실 수 있습니다
```suggestion
    fetch("/Data/ProductList.json")
```

Reply:
우와 처음 알았어요 ㅎㅎ